tests/test_zz_miniapp_kaspi_ofd_v6.py

```
@@ -0,0 +1,155 @@
+    index = (ROOT / "miniapp" / "index.html").read_text(encoding="utf-8")
+
+    assert 'uvicorn.run("korgan.miniapp_api_v6:app"' in launcher
+    assert "miniapp_api_v5:app" not in launcher
+    assert "api_version !== '1.1.0'" in api
+    assert "consultation_ai_receipt_verification !== false" in api
+    assert "consultation_ofd_receipt_verification !== true" in api
+    assert "document_ai_receipt_verification !== false" in api
+    assert "document_ofd_receipt_verification !== true" in api
+    assert "JSON.stringify({ qr_url: qrUrl })" in api
+    assert "BarcodeDetector" in api
+    assert "AI не принимает решение об оплате" in ui
```

Ruff (0.16.2)

[warning] String contains ambiguous `о` (CYRILLIC SMALL LETTER O). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF001)